spec/carthage_cache/project_spec.rb

```diff
@@ -22,6 +22,19 @@
 
   end
 
+  describe "#archive_path" do
+    
+    it "returns the path of the archive for the current Cartfile.resolved file" do
+      project = CarthageCache::Project.new(FIXTURE_PATH, cache_dir_name, "path", terminal, TMP_PATH, MockSwiftVersionResolver.new)
```

Comment:
Can you override `project` using subject inside the `describe` block? I think it would read better.

```ruby
describe "#archive_path" do

  subject(:project) { CarthageCache::Project.new(FIXTURE_PATH, cache_dir_name, "path", terminal, TMP_PATH, MockSwiftVersionResolver.new) }

end
```

Not sure if this is possible